src/routes/display/+layout.svelte

```diff
@@ -43,4 +43,10 @@
     <a href="https://kaffediem.asvg.no" class="font-bold text-accent">kaffediem.asvg.no</a>
     <span>for å bestille</span>
   </div>
+
+</div>
+<div class="absolute bottom-0 left-0 hidden md:flex w-screen">
+  <div class="ml-2 mr-2 flex flex-col items-center justify-center py-1 text-xl text-neutral flex-grow text-center h-48">
+    <p>Laget av elevene fra ITK :) </p>
```

Comment:
Kunne det stått heller noe slikt som
```suggestion
    <p>Laget av elever på skolen :) </p>
```